modules/jooby-akka/src/main/java/org/jooby/akka/Akka.java

```diff
@@ -205,6 +205,7 @@
 
 import static java.util.Objects.requireNonNull;
 
+import com.sun.istack.internal.NotNull;
```

Comment:
use `javax.annotaion.Nonnull` and avoid use of internal classes

Reply:
Hi, it seems the JSR-305 import in the POM was missing. I added it globally in the parent POM.

Any intention from your side regarding such global POM imports? A lot of imports could be moved from modules to parent POM (e.g. PowerMock, etc.). Any thoughts on that?

Reply:
IntelliJ did a Fallback on NotNull in a strange way...fixed it

Reply:
We shouldn't add global dependencies to parent pom.xml

In core, this was handled it like:

```
<dependency>
      <groupId>com.google.code.findbugs</groupId>
      <artifactId>jsr305</artifactId>
      <scope>provided</scope>
    </dependency>
```

Reply:
This does work, if you add Java8EE manually to your IDE (mine hasn't this libs). Maybe I have a configiguration problem here? Any hints would be appreciated?